### PR TITLE
Nimakms/first review

### PR DESCRIPTION
**Pull request by nimakms**
Updates to the grouping of the instructions
Minor changes to template

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,56 +4,82 @@ platforms: dotnet
...
-2. Add a redirect URI that matches the Token Vault that will be created in step 2. (https://tokenvaultname.westcentralus.tokenvault.azure.net) where <tokenvaultname> matches the name specified when deploying below.
+1. Go to the [Dropbox developer portal](https://www.dropbox.com/developers).
+2. **Sign in** using the link on top-right of the site. **[Sign up](https://www.dropbox.com/register)** if you do not have an account already.
+3. [Create a new app](https://www.dropbox.com/developers/apps/create), choose **Dropbox API**, **Full Dropbox** access, and create a unique name for your app.
+4. Record the **App key** and **App secret** vaules for future use.
+5. Set the redirect URI to `https://[token-vault-name].westcentralus.tokenvault.azure.net/redirect` where `[token-vault-name]` is the name of your token vault, that you will create in the next step.
 
 ## Step 2: Create an App Service with a Managed Service Identity (MSI)
-<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fjoerob-msft%2Fapp-service-msi-tokenvault-dotnet%2Fmaster%2Fazuredeploy.json" target="_blank">
```
Comment (nimakms) [inline README.md]: Uncomment.

File: README.md [hunk 2]
```diff
@@ -4,56 +4,82 @@ platforms: dotnet
...
-## Step 1: Create a Dropbox App to integrate with
-Log in to <a href="https://www.dropbox.com/developers/apps/create" target="_blank">Dropbox</a> and create an API App that this sample App Service integrates with.
+## Step 1: Create a Dropbox developer app
 
-1. Note the **App Key** and **App Secret** from the App Settings page.
-2. Add a redirect URI that matches the Token Vault that will be created in step 2. (https://tokenvaultname.westcentralus.tokenvault.azure.net) where <tokenvaultname> matches the name specified when deploying below.
+1. Go to the [Dropbox developer portal](https://www.dropbox.com/developers).
+2. **Sign in** using the link on top-right of the site. **[Sign up](https://www.dropbox.com/register)** if you do not have an account already.
+3. [Create a new app](https://www.dropbox.com/developers/apps/create), choose **Dropbox API**, **Full Dropbox** access, and create a unique name for your app.
+4. Record the **App key** and **App secret** vaules for future use.
```
Comment (joerob-msft) [inline README.md]: values (sp)

File: README.md [hunk 3]
```diff
@@ -90,15 +116,10 @@ public async System.Threading.Tasks.Task<ActionResult> Index()
         return View();
```
Comment (joerob-msft) [inline README.md]: Can you update this to match the code in homecontroller?

**Review by joerob-msft — APPROVED** [review]

**Comment by nimakms** [comment]
Updating instructions to README.md, restructuring scenarios, and adding relevant details.

**Pull request merged**